be/src/exprs/agg/aggregate_factory.cpp

```diff
@@ -449,10 +570,10 @@ class AggregateFuncResolver {
             if (name == "count") {
                 return AggregateFactory::MakeCountNullableAggregateFunction<IsWindowFunc>();
             } else if (name == "sum") {
-                AggregateFunctionPtr sum = AggregateFactory::MakeSumAggregateFunction<ArgPT>();
+                auto sum = AggregateFactory::MakeSumAggregateFunction<ArgPT>();
                 using ResultType = RunTimeCppType<SumResultPT<ArgPT>>;
-                return AggregateFactory::MakeNullableAggregateFunctionUnary<SumAggregateState<ResultType>,
-                                                                            IsWindowFunc>(sum);
+                return AggregateFactory::MakeNullableAggregateFunctionUnary<SumAggregateState<ResultType>, IsWindowFunc,
+                                                                            true>(sum);
```

Comment:
IngoreNull = true?

Reply:
Maybe we should refactor the template here

Reply:
> IngoreNull = true?

We can just use the default values, so I will remove it。